### ISSUE TITLE
does not detect while/until modifier

### ISSUE THREAD

**Issue by eqv** [rubocop/rubocop#215]
Hi

``` ruby
# encoding: utf-8
puts 'hi' while gets
```

result:

``` bash
$ rubocop -v   
0.8.0
$ rubocop test.rb
== test.rb ==
C:  2: Favor modifier while/until usage when you have a single-line body.

1 file inspected, 1 offence detected
```

Rubocop ignores that i indeed switched to the modifier form and continues to complain
